translation/beyondtheveil/zh_cn.lang

```diff
@@ -0,0 +1,1110 @@
+research.voidDream.text=梦见虚空。
+
+research.FIRSTCONTACT.title=接触不可知者
+research.FIRSTCONTACT.stage.0=如果所有梦境的确都会经行虚空，那么很可能我不是唯一追寻与虚空相关的知识的梦者。<BR>是时候揭开真相前的帷幕了。我去接触其他梦者的时刻已经到来。无论有多大的风险，无论他们有多么杰出、多么强大，无论他们抱怀着怎样的意图。<BR>无论他们可能有多么不同，哪怕事实证明他们与我完全异族。<BR>无论他们来自何方。无论他们来自什么世界，什么时间，什么现实。<BR>现在一切都无所谓了。这可能是我研究的终点。如果事情必然如此，那么不如从命，而我只希望这些笔记能被证实对后人是有用的。<BR>我必须梦见悚异，这需要在虚空之物的影响下完成。我必须修理双耳，以听到我被告知的一切。我将竭力聆听。并且准备好迎接前所未有的长梦。
+research.FIRSTCONTACT.stage.1=我仁慈的神，在我写下这些文字时请不要让我的心智崩溃。<BR>一个声音。它来自海洋。我并非听到了它，而是感受到了它。它通过梦境触及我身。<BR>它并未对我出言，而是直接穿过了我，就好像它根本没注意到我在那里，在我的梦中。<BR>但我现在知道我必须做什么了。这一想法印在了我的脑海之中。<BR>我必须找到一个地方，在那里虚空的邃暗触碰着万物的表面，如此一来我也许能亲自目睹并接触它。<BR>但是，说实话，我不知道是否还应该继续我的求知之旅......<BR><BR><BR>慈悲的神啊，请怜悯我。<BR>不要让我溺亡其中。
+research.eldritchDream.text=梦见悚异。
+
+research.STRONGHOLD.title=寻找要塞
+research.STRONGHOLD.stage.0=悚异之梦与末地有关，这没什么可惊奇的，毕竟它与虚空的关联如此密切。<BR>未被虚空之梦加强时，悚异之梦将告诉我一个要塞的确切坐标。
+
+research.DARKDREAMS.title=黑暗之梦
+research.DARKDREAMS.stage.0=在进入大海之前，我能感到沙子，它们在我的......我的梦境之下。颜色深沉、黏滑的暗沙，呈现出虚空的颜色。<BR>确实有报告提及潮湿、昏黑的沙滩，更进一步还有流言称这些“虚芜”（Voided）的生物群系定居着一小群深居简出的渔民。<BR>我想知道他们是否能为我的求知之途提供线索。无论如何，我现在的目标是梦见黑暗。这将是一段全新的、令人印象深刻的记忆。
+research.DARKDREAMS.stage.1=这效果远非完美所能形容，黑暗之梦在寻找有限范围内的虚芜生物群系时发挥了巨大的作用。<BR>然而，我害怕我会我失望。一捧黑沙真的能带给我启迪吗？我不这么认为。<BR>或许我应该转而集中精力寻找传言中出现于这些地方的旧渔村。<BR>我很清楚并非所有的虚芜生物群系都有村落，所以我的研究可能要花费很长时间。不过我倒是可以自我安慰，想想我的旅程中能发现多少世界上的新事物。<BR>但我仍心存疑虑，这些传言有多大可能是真实的，又有多大可能根本毫无价值，仅仅是一个卑鄙的谎言？<BR>在了解我们世界之外的彼端是何其广远，比起眼前的表象是何其深邃之后...我陷入了令人痛苦的疑云之中。
+research.darknessDream.text=梦见黑暗。
+
+research.FISHINGHAMLET.title=旧渔村
+research.FISHINGHAMLET.stage.0=我需要真相。我受够了谎言。<BR>必然是有一道幻觉织就的帷幕遮挡了所有的人的心智。我所见的一切，皆为谎言。<BR>§l只有谎言。§r<BR>我必须揭开这道帷幕。我必须看到屏障后的彼端。<BR>无论旧渔村是解开终极奥秘的关键，或者仅仅是宏大谜题的一个碎片，我们终会知晓。
+research.FISHINGHAMLET.stage.1=这不可能...<BR>所有的工作，所有的研究...把我引到了这个地方？！<BR>到一群堕落者之中？！到一群排外的蠢货之中？！？<BR>不不不，我绝不能就此停止。我能揭开帷幕。我知道我能。我确信如此。我坚信不疑。<BR>我需要和灯塔看守谈谈。我希望比起他的居民同伴，他能表现得更加明智。
+research.FISHINGHAMLET.stage.2=看看大海...看看大海...
+research.FindHamlet.text=定位一个旧渔村。
+research.LHKeeper.text=与灯塔看守交谈。
+
+research.IDOL.title=共鸣
+research.IDOL.stage.0=有时候，我们会自以为神。万能的、无上的神，整个世界必须向我们俯首称臣。神，是造物主，能重塑整个世界。<BR>但我们就好像是易受哄骗的愚人。<BR>大海。<BR>无边，永恒，莫测。它真的在乎我们做了什么吗？当它让高大雄伟的建筑显得无比渺小，当它的年寿超越我们最为强劲坚固的巨型装置时，它真的会在乎吗？<BR>大海即是无限的象征，前所未有的接近无限，时刻提醒着我们意识到自己的§l微不足道§r。<BR>大海，它具有着令旧渔村居民无比神往的广远，是§l不可知的§r。我们可以费尽终生冀望研究它的每一个缝隙和角落，然而这一切都是徒劳，因为在它无边无际的领域以及它变化多端的混沌自然之中，居于其中的事物...或者居于其底的事物...往往会逃过我们的视线。除非它们主动来寻找我们。<BR>没错，大海一定就是关键。在它无边的、不可知的无限之中，§l必定§r隐藏着我们正在寻找的一切。就如同某种门户，打开这扇门就能看到帷幕彼端。在大海无底的深渊中，我们一定要找到裂隙。<BR>水§l散发§r着悚异的气息，尤其是那个渔村中的古怪堕落者所崇拜的某种...某种...<BR>没错。某种...当然，他们绝非堕落者，因为他们远比§l我们§r更早地找到了解决方法。<BR>如果他们的神祇真的是帷幕彼端的展现形式，将真相从我们所有人眼前隐藏，如果他们对梦和海的狂热是出于与我们共同的天性...那么我们必须寻求共鸣。<BR>与那伟大长梦者共鸣。
+research.IDOL.stage.1=我谦卑地向你发誓献上我的生命，伟大长梦者。<BR>我否定过往的所有信仰，所有我曾深信不疑的神灵。<BR>我对你而言微不足道。大海，是你在主世界的化身，是你的象征之物。<BR>你对我而言是不可知的。遍布虚空的虚无，即是你高深莫测的本质。<BR>你所深藏的彼端万象皆存。那即是真相，被帷幕所藏。<BR>赐予我们知识，如此我们方能揭穿一切谎言。<BR>赐予我们力量，如此我们方能在迎接真相时幸存。<BR>赐予我们勇气...如此我们方能聆听你的神圣之音。
```

Comment:
|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
research.IDOL.stage.1=我谦卑地向你发誓献上我的生命，伟大长梦者。<BR>我否定过往的所有信仰，所有我曾深信不疑的神灵。<BR>我对你而言微不足道。大海，是你在主世界的化身，是你的象征之物。<BR>你对我而言是不可知的。那遍布虚空的虚无，即是你高深莫测的本质。<BR>你所深藏的彼端万象皆存。那即是真相，被帷幕所藏。<BR>赐予我们知识，如此我们方能揭穿一切谎言。<BR>赐予我们力量，如此我们方能在迎接真相时幸存。<BR>赐予我们勇气...如此我们方能聆听你的神圣之音。
```

Reply:
leave it.